samples/embedded-auth-with-sdk/src/main/java/com/okta/spring/example/helpers/Util.java

```diff
@@ -73,4 +75,18 @@ public static void setProceedContextForPoll(final HttpSession session, final Pro
     public static ProceedContext getProceedContextForPoll(final HttpSession session) {
         return (ProceedContext) session.getAttribute("proceedContextForPoll");
     }
+
+    /**
+     * Construct RequestContext object with random `X-Device-Token` header value.
+     *
+     * @return request context object
+     */
+    public static RequestContext constructRequestContext() {
+        final RequestContext requestContext = new RequestContext();
+        // generate random str 32 char long
+        final UUID randomUUID = UUID.randomUUID();
+        final String randomString = randomUUID.toString().replaceAll("-", "");
```

Comment:
Rumor has it that this field is limited to 32 characters on the server, and the need to strip the `-` are a result of that.  While it isn't a problem to strip these here.  It does add more burden on the developer, many UUID implementations add the dashes in a `toString()`.  The server should just be updated to increase this field size.  If nothing else to prevent this type of confusion to the developer.

Can you report this to the correct team?

Reply:
Initiated an internal conversation with the owning team.